mathml/presentation-markup/mrow/mrow-fallback.html

```diff
@@ -11,6 +11,9 @@
 <script src="/mathml/support/fonts.js"></script>
 <script src="/mathml/support/layout-comparison.js"></script>
 <script src="/mathml/support/mathml-fragments.js"></script>
+<style>
+  mfrac { padding-inline: 0; }
```

Comment:
Can you please add a comment explaining this style change for someone reading the test (see https://github.com/web-platform-tests/wpt/blob/d90f52b5215dca9fd334314b7353a4b57b4719c9/mathml/relations/css-styling/size-containment-001.tentative.html#L12 for example)?

Reply:
Added